src/pages/ChatPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import './ChatPage.css';
+
+const ChatPage = ({location, history}) => {
+    const name = location.state;
+    const [message, setMessage] = useState([]);
+    const userText=['안녕하세요', '샘플 데이터입니다', '반가워요', '날씨가 어때요?', '감사합니다', '좋은하루 되세요', '사랑해요ㅎ.ㅎ'];
+    const [value, setValue] = useState('');
+    const onChange = (event) => {
```

Comment:
```suggestion
    const onChangeInputMessage = (event) => {
```

변수/함수 네이밍 고민을 해보시는게 좋을 것 같아요.
나중에 코드가 무척 길어지면... onChange가 무엇이 변경되었을 때 호출 되는건지 알기 위해선 많이 피곤해집니다^^